plugins/source/aws/codegen/recipes/mwaa.go

```diff
@@ -0,0 +1,36 @@
+package recipes
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/mwaa/types"
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func MWAAResources() []*Resource {
+	resources := []*Resource{
+		{
+			SubService:          "environments",
+			Description:         "https://docs.aws.amazon.com/mwaa/latest/API/API_Environment.html",
+			Struct:              &types.Environment{},
+			SkipFields:          []string{"Arn"},
+			PreResourceResolver: "getEnvironment",
+			ExtraColumns: append(
+				defaultRegionalColumns,
+				[]codegen.ColumnDefinition{
+					{
+						Name:     "arn",
+						Type:     schema.TypeString,
+						Resolver: `schema.PathResolver("Arn")`,
+						Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+					},
+				}...),
+		},
+	}
+
+	// set default values
+	for _, r := range resources {
+		r.Service = "mwaa"
```

Comment:
`mwaa` or `airflow` though? if you look at the line below the service id is apparently "airflow" but the Go package name is "mwaa. So it could be either "aws_mwaa_environments" as it is, or "aws_airflow_environments". 🤔